module/VuFindTheme/src/VuFindTheme/View/Helper/HeadThemeResources.php

```diff
@@ -39,144 +39,18 @@
 class HeadThemeResources extends \Laminas\View\Helper\AbstractHelper
 {
     /**
-     * Theme resource container
+     * Set up items based on contents of theme resource container.
      *
-     * @var \VuFindTheme\ResourceContainer
-     */
-    protected $container;
-
-    /**
-     * Constructor
-     *
-     * @param \VuFindTheme\ResourceContainer $container Theme resource container
-     */
-    public function __construct(\VuFindTheme\ResourceContainer $container)
-    {
-        $this->container = $container;
-    }
-
-    /**
-     * Set up header items based on contents of theme resource container.
+     * @deprecated Deprecated, use ThemeResources.
      *
      * @return void
      */
     public function __invoke()
     {
-        // Add various types of content to the header:
-        $this->addMetaTags();
-        $this->addLinks();
-        $this->addScripts();
-    }
-
-    /**
-     * Given a colon-delimited configuration string, break it apart, making sure
-     * that URLs in the first position are not inappropriately split.
-     *
-     * @param string $current Setting to parse
-     *
-     * @return array
-     */
-    protected function parseSetting($current)
-    {
-        $parts = explode(':', $current);
-        // Special case: don't explode URLs:
-        if (($parts[0] === 'http' || $parts[0] === 'https')
-            && '//' === substr($parts[1], 0, 2)
-        ) {
-            $protocol = array_shift($parts);
-            $parts[0] = $protocol . ':' . $parts[0];
-        }
-        return $parts;
-    }
-
-    /**
-     * Add meta tags to header.
-     *
-     * @return void
-     */
-    protected function addMetaTags()
-    {
-        // Set up encoding:
-        $headMeta = $this->getView()->plugin('headMeta');
-        $headMeta()->prependHttpEquiv(
-            'Content-Type', 'text/html; charset=' . $this->container->getEncoding()
+        trigger_error(
```

Comment:
It would make sense to give this `E_USER_DEPRECATED` as the second parameter, to raise a more appropriate error level.